server/src/main/java/org/elasticsearch/common/lucene/Lucene.java

```diff
@@ -84,6 +84,10 @@ public static Version parseVersion(@Nullable String version, Version defaultVers
         }
     }
 
+    public static org.apache.lucene.util.Version luceneVersion(int major, int minor, int bugfix) {
+        return org.apache.lucene.util.Version.fromBits(major, minor, bugfix);
+    }
+
```

Comment:
What's the reason to wrap this instead of using `fromBits` directly?

Reply:
It means that the `Version` class has a bunch of references to `Lucene.luceneVersion()` rather than `org.apache.lucene.util.Version.fromBits()` which just looked less cluttered to me.